src/components/Task/Task.css

```diff
@@ -0,0 +1,12 @@
+.list-item {
+    background-color: azure;
+}
+.TASK_PINNED {
+   background-color: lightgray
+}
+
+.TASK_ARCHIVED {
+     background-color: lightcoral;
+     text-decoration: line-through;
```

Comment:
should use scss instead of css

Reply:
I haven't used scss before how do you recommend me to install it? npm or should I look for a faster option?

Reply:
change the file name Task.css to Task.scss, nan already set it up

Reply:
Done